changelogs/fragments/174-return-value-fix.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+        - digital_ocean_kubernetes - fix return value consistency (https://github.com/ansible-collections/community.digitalocean/issues/174)
```

Comment:
Nit: These should end with a period `.` and we generally indent list items with two (2) spaces.